### DISCUSSION TITLE
change header search and column count text

### DISCUSSION THREAD

**Discussion by amindeist** [rappasoft/laravel-livewire-tables#901] [Q&A]
hi
how i can to change header search and column count text to my language?

**Comment by MazenRex** [comment]
hi
you need to publish the translations and edit them in your own language by
```
php artisan vendor:publish --provider="Rappasoft\LaravelLivewireTables\LaravelLivewireTablesServiceProvider" --tag=livewire-tables-translations
```
for more info the [Documentation](https://rappasoft.com/docs/laravel-livewire-tables/v2/start/configuration).
